parts/k8s/addons/antrea.yaml

```diff
@@ -332,9 +372,26 @@ spec:
         key: node-role.kubernetes.io/master
       volumes:
       - configMap:
-          name: antrea-config-48gttf992h
+          name: antrea-config-476k567258
         name: antrea-config
 ---
+---
```

Comment:
Cluster deployments are failing on this PR against this test config:

https://github.com/Azure/aks-engine/blob/master/test/e2e/test_cluster_configs/network_policy/antrea.json

There is an extra `---` above, perhaps that's failing to be parsed by `kube-addon-manager`.

Reply:
Verified the latest change locally by running e2e tests and it passed. Please re-run e2e test for checkin. Thanks!